### PR TITLE
Add committer status for myself

### PR DESCRIPTION
**Pull request by xingri**
#### 8732eb4f0cc418237e1e625f91e36f9926f98329
<pre>
Add committer status for myself
No bug applicable.

Unreviewed commit, adding committer status/data for myself to contributors.json.

* metadata/contributors.json:

Canonical link: <a href="https://commits.webkit.org/268839@main">https://commits.webkit.org/268839@main</a>
</pre>

https://github.com/WebKit/WebKit/commit/ecf3a8baad88bf1d641318b760e289d84d8cc805

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/20801 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/21208 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/21868 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/22690 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/19393 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/24449 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/21385 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/20697 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/21022 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/20802 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/18064 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/23544 "Built successfully") | 
| [✅ 🧪 webkitpy](https://ews-build.webkit.org/#/builders/6/builds/20987 "Passed tests") | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/17967 "Passed tests") | | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/25182 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/19046 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/19054 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/23083 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/19643 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/16686 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/18877 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/4995 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/23208 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/19461 "Built successfully") | | |

### CODE REVIEW

**Comment by JonWBedard** [comment]
@xingri, I don't see a committer nomination for you or Joone, per the policy posted on https://webkit.org/commit-and-review-policy/#criteria-for-committers, you would need to be nominated by a reviewer on webkit-reviewers@lists.webkit.org and have 2 reviewers second that nomination.

That being said, what _can_ be done right now is to give you and Joone the ability to modify pull requests (as you only need be a contributor for that), and what's missing from `metadata/contributors.json` for that, at the moment, is a `github` entry for both of you. Without that `github` entry, matching someone to their GitHub identity is an intractable problem.

**Comment by MenloDorian** [comment]
I know that Byungseon Shin (xingri) was previously a committer who became inactive.  On Sep 6, 2023, he landed a substantive patch https://commits.webkit.org/267677@main, and requested via committer email list to be granted his committer privileges again.  According to WebKit committer reviewer policy, I believe he has met the requirement to be granted committer privileges.

As for Joone Hur, the last substantive patch by Joone was back on Aug 27, 2021.  I also don't know if Joone previously had committer privileges, though there were some commits by Joone back in history many years ago.

Byungseon, please drop Joone's change from this PR.  Separately, Joone will need to show that he/she previously had committer privilege and if so, also need to write at least one substantive patch in order to regain the privilege.

**Comment by MenloDorian** [comment]
Byungseon, Joone will also need to request to be re-granted committer privileges via the committers' email list after he/she lands a substantive patch.  Thanks.

**Comment by MenloDorian** [comment]
According to `git blame 25afdaadd4f14b3bae59b22bf07f6e405345338e Tools/Scripts/webkitpy/common/config/contributors.json`, I see that Joone was indeed a committer once upon a time.  So, he/she just needs to follow the steps outlined in the WebKit committer reviewer policy to regain committer privilege if so desired.

**Comment by JonWBedard** [comment]
As a side note, in the modern GitHub world, your committer status is meaningless without a GitHub username, that's what `Merge-Queue` is using to know that the user which added a specific label is permitted to land changed.

**Comment by xingri** [comment]
@JonWBedard  and @MenloDorian  Thank you for the comments. I have just updated commit to include myself only.

**Comment by JonWBedard** [comment]
> @JonWBedard and @MenloDorian Thank you for the comments. I have just updated commit to include myself only.

I'm happy to land this as is, but @xingr , without a `github` entry, your committer status won't actually let you land changes, since it's your GitHub user that will be adding the `merge-queue` label. I suspect you are interested in being able to land changes yourself, so I think you want to add your GitHub username to your contributors.json entry.

**Comment by xingri** [comment]
> > @JonWBedard and @MenloDorian Thank you for the comments. I have just updated commit to include myself only.
> 
> I'm happy to land this as is, but @xingr , without a `github` entry, your committer status won't actually let you land changes, since it's your GitHub user that will be adding the `merge-queue` label. I suspect you are interested in being able to land changes yourself, so I think you want to add your GitHub username to your contributors.json entry.

@JonWBedard  No I understand what you are meaning. I believe you are referring me as @xingr instead of @xingri  and that's why you couldn't find my github entry. Hope you can find me by @xingri

File: metadata/contributors.json [hunk 1]
```diff
@@ -1369,7 +1369,8 @@
       "name" : "Byungseon(Sun) Shin",
       "nicks" : [
          "xingri"
-      ]
+      ],
+      "status" : "committer"
```
Comment (aprotyas) [inline metadata/contributors.json:1374]:
```suggestion
      ],
      "github" : "xingri",
      "status" : "committer"
```

**Review by aprotyas — COMMENTED** [review]
What @JonWBedard meant was that you need to add a `"github" : "<username>"` entry for yourself.

File: metadata/contributors.json (hunk 1 above)
Comment (xingri) [reply metadata/contributors.json:1374]: @aprotyas  Thanks you for the feedback. Now I fully understood.

**Comment by xingri** [comment]
> @xingri - Can you squash your both commits as one?

@Ahmad-S792 Sure. will do it shortly.

**Comment by aprotyas** [comment]
(In addition to squashing, the github entry needs to move up to maintain alphabetical ordering of the keys, as pointed out by the style guide)

**Comment by xingri** [comment]
> (In addition to squashing, the github entry needs to move up to maintain alphabetical ordering of the keys, as pointed out by the style guide)

@aprotyas  Thanks. I just updated the change.

**Comment by aprotyas** [comment]
Thanks for the patience @xingri. Merging!

**Comment by xingri** [comment]
> Thanks for the patience @xingri. Merging!

Thanks for the merging the change @aprotyas .

**Pull request merged**